guides/v2.2/cloud/env/variables-deploy.md

```diff
@@ -129,7 +129,7 @@ stage:
 -  **Default**—_Not set_
 -  **Version**—Magento 2.1.4 and later
 
-{{ .bs-callout-warning }}
+{{: .bs-callout-warning }}
```

Comment:
Only one set of curly braces is required; not two.

**Incorrect:**

```
{{:.bs-callout-warning}}
```

**Correct:**

```
{:.bs-callout-warning}
```

Reply:
Thank you for letting me know. This is fixed.